### DISCUSSION TITLE
Is there a way to hide the AlDente icon from the Dock?

### DISCUSSION THREAD

**Discussion by oliwiergawlik** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1151] [Q&A] [answered]
Hey, I would like to hide AlDente app icon from my dock so that it only shows up in the taskbar like other applications that run in the background.
It frustrates me and clutter dock and command+tab window.

Is there any way to do so?

**Comment by DamonHD** [comment, marked as answer]
I have disabled the dock icon for my machine under the Appearance tab (V1.24.1 paid).

**Reply by oliwiergawlik** [reply]
<img width="429" alt="image" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/90700152/6bc144fe-ed2f-4b1f-819b-7af569792b57">

Of course there is such an option, I just didn't notice it...
Thanks!
